templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -369,7 +391,7 @@ contents:
       ovs-vsctl --timeout=30 --if-exists del-br br-int -- --if-exists del-br br-local -- --if-exists del-br br-ex
 
       if [[ -n "$iface" ]]; then
-        nmcli device connect $iface
+        nmcli c up ifname $iface
```

Comment:
whats the reason for this change? will this work if the device was disconnected?

Reply:
For some reason, when config is backed up by ifcfg files and we remove connection profiles, the corresponding device disappears and then we cannot connect by device. So at the end is more reliable to connect by profile and fortunately we can do this in combination with the device name as well.